pylint_odoo/checkers/no_modules.py

```diff
@@ -232,6 +257,18 @@ def visit_callfunc(self, node):
                     self.add_message('translation-field',
                                      node=argument_aux)
 
+        # Check if cr.execute() has an sql-injection
+        if node.as_string().lower().startswith('cr.execute(') or \
+                ("cr.execute(" or "_cr.execute(") in node.as_string():
```

Comment:
You can just use:
`if 'cr.execute' in node.as_string():`
This work for self.env.cr.execute, and self._cr.execute and so on